Mathlib/Data/Finsupp/Fin.lean

```diff
@@ -4,6 +4,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Ivan Sadofschi Costa
 -/
 import Mathlib.Data.Finsupp.Defs
+import Mathlib.Algebra.BigOperators.Finsupp
+import Mathlib.Algebra.BigOperators.Fin
```

Comment:
I'm not convinced this is the right way around for these imports to go. I think adding the lemma to `BigOperators.Fin` or `BigOperators.Finsupp` would make more sense.

Reply:
Well we've got `Fin`, `Finsupp`, and `BigOperators`, I guess it's pick two.